app/views/catalog/_resource_attributes_ephemera_folder.html.erb

```diff
@@ -0,0 +1,28 @@
+<div id="attributes">
```

Comment:
Can we avoid some of these duplicate views?  Could we make one of them `_resource_attributes_default.html.erb`? Could we have one copy and then have this view and the other one call out to it with `render partial` ?